src/iget.cpp

```diff
@@ -159,7 +159,7 @@ usage( FILE* _fout ) {
         "Large files are defined by the 'maximum_size_for_single_buffer_in_megabytes'",
         "advanced configuration setting.",
         " ",
-        "The -Q option specifies the use of the RBUDP transfer mechanism which uses",
+        "The -Q option [Deprecated] specifies the use of the RBUDP transfer mechanism which uses",
```

Comment:
Same here - perhaps make the deprecation note a separate sentence?

Reply:
done